### PR TITLE
Automate documentation of table associations

### PR DESCRIPTION
**Pull request by yoomlam**
### Description
Create visualizations for table associations.

The `svg` files are committed (since they are text and can be diff'd) and linked from https://github.com/department-of-veterans-affairs/caseflow/wiki/Caseflow-Database-Schema-Documentation (when a diagram is clicked).

The `png` images are manually copied and pasted to https://github.com/department-of-veterans-affairs/caseflow/wiki/Caseflow-Database-Schema-Documentation

### Acceptance Criteria
- [ ] Code compiles correctly

### Testing Plan
* Run `make doc` or `make doc-schema-caseflow` directly.
* Examine images `docs/schema/caseflow-*.png` and `svg` files in browser. Descriptions of the diagrams are provided at https://github.com/department-of-veterans-affairs/caseflow/wiki/Caseflow-Database-Schema-Documentation.

### Code Documentation Updates
- [x] Add or update code comments at the top of the class, module, and/or component.

### CODE REVIEW

**Comment by yoomlam** [comment]
## caseflow-subclasses
image

## caseflow-belongs_to_erd
image

## caseflow-belongs_to_erd-subclasses
image

File: lib/helpers/association_wrapper.rb [hunk 1]
```diff
@@ -6,68 +6,80 @@
 ##
 
 class AssocationWrapper
```
Comment (yoomlam) [inline lib/helpers/association_wrapper.rb:8]: Did a lot of refactoring to make this class more readable. Ignore the diffs and review the refactored class by itself.

File: lib/tasks/doc.rake [hunk 2]
```diff
@@ -3,102 +3,204 @@
...
-            attr.name,
-            attr.type_description,
-            pretty_boolean(attr.mandatory?),
-            pretty_boolean(attr.primary_key?),
-            pretty_boolean(attr.foreign_key?),
-            pretty_boolean(attr.unique?),
-            pretty_boolean(indexed?(table_name, attr.name)),
-            comment
-          ]
-        end
```
Comment (yoomlam) [inline lib/tasks/doc.rake:100]: Refactored be within the `csv` namespace.

**Comment by yoomlam** [comment]
improved edge label position
image

**Review by lowellrex — APPROVED** [review]
This looks great! Thanks @yoomlam

**Pull request merged**